tools/run_tests/run_interop_tests.py

```diff
@@ -630,17 +635,29 @@ def cloud_to_cloud_jobspec(language, test_case, server_name, server_host,
       '--use_tls=%s' % ('false' if insecure else 'true'),
       '--use_test_ca=true',
   ]
+
+  client_test_case = test_case
+  if test_case in _HTTP2_SERVER_TEST_CASES_THAT_USE_GRPC_CLIENTS:
+    client_test_case = _GRPC_CLIENT_TEST_CASES_FOR_HTTP2_SERVER_TEST_CASES[test_case]
+
   common_options = [
-      '--test_case=%s' % test_case,
+      '--test_case=%s' % client_test_case,
       '--server_host=%s' % server_host,
   ]
-  if test_case in _HTTP2_BADSERVER_TEST_CASES:
-    # We are running the http2_badserver_interop test. Adjust command line accordingly.
-    offset = sorted(_HTTP2_BADSERVER_TEST_CASES).index(test_case)
+
+  if test_case in _HTTP2_SERVER_TEST_CASES:
+    # We are running the http2_server_interop test. Adjust command line accordingly.
+    offset = sorted(_HTTP2_SERVER_TEST_CASES).index(test_case)
     client_options = common_options + ['--server_port=%s' %
```

Comment:
It seems the negative cases are left out. Maybe that's why they are not executed?